src/components/SubpageUpload/http-common.js

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
```

Comment:
Put content of this file to the PageUpload and remove this file

Reply:
I dont know how to do this in a smart way, since this is how it is explained by the offical documentation.